frontend/components/copytoclipboard.js

```diff
@@ -0,0 +1,36 @@
+import { useState }  from "react";
```

Comment:
import React, { useState }  from "react";

Reply:
I think the new React doesn't require to be imported :)